setup.cfg

```diff
@@ -12,6 +12,7 @@ exclude_lines =
     pragma: no cover
     raise NotImplementedError
     return NotImplemented
+    if TYPE_CHECKING:
```

Comment:
Is it expected?

Reply:
This was added because `codecov` treated the code behind `if` as not covered. Now it is ignoring it.